src/billing/sdk/instructions/admin/init-system-vault.instruction.ts

```
@@ -0,0 +1,33 @@
+import { Connection, PublicKey, TransactionInstruction } from '@solana/web3.js';
+import { getProgram } from '../../utils/program';
+
+interface InitSystemVaultAccounts {
+  operator: PublicKey;
+  systemAuthority: PublicKey;
+  systemProgram: PublicKey;
+}
+
+interface InitSystemVaultParams {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)